src/logic/Game/Game.tsx

```diff
@@ -0,0 +1,45 @@
+import { Props } from '../Entity/types';
+import { Tile } from '../Tile/Tile';
+
+
+export default class Game {
```

Comment:
❗️
Кажется, закралась лишняя строка
```suggestion
import { Tile } from '../Tile/Tile';

export default class Game {
```

Reply:
Убрал